mungegithub/mungers/submit-queue.go

```diff
@@ -466,6 +506,10 @@ func (sq *SubmitQueue) e2eStable(aboutToMerge bool) bool {
 	wentUnstable := false
 
 	stable, ignorableFlakes := sq.e2e.GCSBasedStable()
+	if stable && sq.emergencyMergeStop() {
+		stable = false
+		glog.Errorf("E2E counts as not stable because an emergency merge stop is in progress.")
```

Comment:
I will adjust.

On Tue, Jun 14, 2016 at 10:46 AM, Eric Paris <notifications@github.com>
wrote:

> In mungegithub/mungers/submit-queue.go
> <https://github.com/kubernetes/contrib/pull/1204#discussion_r67020306>:
>
> > @@ -466,6 +506,10 @@ func (sq *SubmitQueue) e2eStable(aboutToMerge bool) bool {
> >  	wentUnstable := false
> >
> >  	stable, ignorableFlakes := sq.e2e.GCSBasedStable()
> > +	if stable && sq.emergencyMergeStop() {
> > +		stable = false
> > +		glog.Errorf("E2E counts as not stable because an emergency merge stop is in progress.")
>
> I'm worried this is going to log as fast as the CPU will let it log if the
> tests are all green, there is something in the queue, but it has been
> stopped...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1204/files/6efcf736314a4bba04c191182a3d6675a275b593#r67020306>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAngllSEEzWmGhqjL-hyWqDbtnRO8tJvks5qLujhgaJpZM4I1iR4>
> .
>